pilot/pkg/util/sets/string.go

```diff
@@ -139,3 +146,15 @@ func (s Set) Equals(other Set) bool {
 func (s Set) Empty() bool {
 	return len(s) == 0
 }
+
+func (s Set) Set(value string) error {
+	parts := strings.Split(value, ",")
+	for _, part := range parts {
```

Comment:
What's this, i haven't looked at kind of set like this

Reply:
It will lead to misuse.

Reply:
Oh yeah I didn't notice this was in the core code. This should probably just be done in the test code to make sure no one uses this in core code?

Reply:
Agree

Reply:
Like `type setFlag sets.Set` or something +1